cmd/prompt.go

```diff
@@ -136,9 +136,9 @@ var promptDirs []string
 
 // used to init or reset pathCmd flags for gnmic-prompt mode
 func initPromptFlags(cmd *cobra.Command) {
-	cmd.Flags().StringArrayVarP(&promptFiles, "file", "", []string{}, "yang files to get the paths")
-	cmd.Flags().StringArrayVarP(&promptExcluded, "exclude", "", []string{}, "yang modules to be excluded from path generation")
-	cmd.Flags().StringArrayVarP(&promptDirs, "dir", "", []string{}, "directories to search yang includes and imports")
+	cmd.Flags().StringArrayVarP(&promptFiles, "file", "", []string{}, "path to yang files to get auto-completions from")
+	cmd.Flags().StringArrayVarP(&promptExcluded, "exclude", "", []string{}, "yang module names to be excluded from auto-completion generation")
```

Comment:
I would add "... excluded from __path__ auto-completion generation"

Reply:
Initially I on purpose removed the path, as we do (plan to?) extract more than just path information, like types.

But if that causes confusion, lets revert it to having **path** in the auto-completion. To be consistent, the **path** should be added to --file as well